velox/dwio/parquet/tests/ParquetTpchTest.h

```diff
@@ -32,13 +33,14 @@ using namespace facebook::velox::exec;
 using namespace facebook::velox::exec::test;
 using namespace facebook::velox::parquet;
 
-static const int kNumDrivers = 4;
+namespace facebook::velox::exec::test {
+namespace {
 
-class ParquetTpchTest : public testing::Test {
- protected:
+class ParquetTpchTestBase : public testing::Test {
```

Comment:
Is there a reason to have this Base class?
The alternative is to use a single class.

Reply:
I want to use this base class to test other queries on Tpch tables and verify their correctness, specifically the ones that use 'like' operator in Re2FunctionsTest.

Reply:
Can you rename the file to `ParquetTpchTestBase.h`?